packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2708,9 +2708,11 @@ export class ContainerRuntime
 				this.updateDocumentDirtyState(false);
 			}
 
-			this.validateAndProcessRuntimeMessage(messageWithContext, localOpMetadata);
+			if (messageWithContext.isModernRuntimeMessage) {
```

Comment:
Or if the type is in the list of ContainerMessageType.  Otherwise legacy runtime ops (stuck in trailing ops in an old file) will not be processed.

Reply:
If there are no tests that break as-is.... then yikes!  But I'm not sure there are.  @agarwal-navin do the snapshot tests include trailing ops too?  I wonder if there are any from back before the "op" envelope.

Reply:
A good reminder to test the code with E2E tests locally.